components/tracker/src/lib.rs

```diff
@@ -34,9 +34,38 @@ impl Tracker {
     pub fn write_scan_detail(&self, detail_v2: &mut pb::ScanDetailV2) {
         detail_v2.set_rocksdb_block_read_byte(self.metrics.block_read_byte);
         detail_v2.set_rocksdb_block_read_count(self.metrics.block_read_count);
+        detail_v2.set_rocksdb_block_read_nanos(self.metrics.block_read_nanos);
         detail_v2.set_rocksdb_block_cache_hit_count(self.metrics.block_cache_hit_count);
         detail_v2.set_rocksdb_key_skipped_count(self.metrics.internal_key_skipped_count);
         detail_v2.set_rocksdb_delete_skipped_count(self.metrics.deleted_key_skipped_count);
+        detail_v2.set_get_snapshot_nanos(self.metrics.get_snapshot_nanos);
+    }
+
+    pub fn write_write_detail(&self, detail: &mut pb::WriteDetail) {
+        detail.set_store_batch_wait_nanos(self.metrics.wf_batch_wait_nanos);
```

Comment:
Do we need to link the performance map in the comments to explain the relationships?

Reply:
Let me the change some bits of the performance to match the metrics first.